### PR TITLE
Add manifest for GNOME Firmware

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: org.gnome.Firmware.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.gnome.Firmware",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
```
Comment (bilelmoussaoui) [inline org.gnome.Firmware.json]:
```suggestion
  "runtime-version": "3.34",
```
Any reason for not using 3.34?

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.Firmware

File: org.gnome.Firmware.json (hunk 1 above)
Comment (hughsie) [reply org.gnome.Firmware.json]: No reason, but I've not tested it with that runtime version. Give me a few mins and I'll do just that.

Comment (hughsie) [reply org.gnome.Firmware.json]: Seems to work fine with the new version.

**Comment by hughsie** [comment]
Hoisted by my own petard.

**Comment by LeandroStanger** [comment]
bot, build org.gnome.Firmware

**Comment by hughsie** [comment]
bot, build org.gnome.Firmware

**Comment by hughsie** [comment]
@bilelmoussaoui I've committed the AppData files upstream and patched them in the flathub build. Success!

**Comment by bilelmoussaoui** [comment]
I just tried the test build locally and everything seems to work fine but when I try to refresh the metadata I'm getting this. Adding `--share=network` fixes this for me

Screenshot from 2019-09-13 13-46-38

**Comment by hughsie** [comment]
@bilelmoussaoui fixed ^^ good catch.

**Comment by bilelmoussaoui** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Firmware


Thanks!

**Comment by hughsie** [comment]
Thanks @bilelmoussaoui -- that was super quick!

**Pull request closed without merging**